apps/api-example/src/app/index.ts

```diff
@@ -19,6 +19,8 @@ import {Application} from 'express'
 import {DefaultURLAdapter} from '../urlAdapters'
 import {readConfig} from '../readConfig'
 import {MannschaftURLAdapter} from '../urlAdapters/URLAdapter-mannschaft'
+import {TrackingPixelProvider} from '../../../../libs/tracking-pixel/api/src/lib/tracking-pixel-provider/tracking-pixel-provider'
```

Comment:
Imports are not using the library but the file directly

Reply:
Still in draft ;-) dont worry